meta/contao/mobile-page-layout-bundle/de.yml

```diff
@@ -0,0 +1,6 @@
+de:
+    title: Mobiles Seitenlayout für Contao 4.8+
+    description: >
+        In Contao 4.8 wurde die Unterstützung für mobile Seitenlayouts entfernt.
+        Wenn ihre Webseite diese Funktion weiterhin benötigt, installieren Sie diese Erweiterung. Beachten Sie
```

Comment:
Typo: **i**hre -> **I**hre 
(Sollten wir hier vielleicht ebenfalls bei der Du-Form bleiben, wie in der [README](https://github.com/contao/package-metadata/blob/master/docs/de/README.md) empfohlen?)

Typo: Nach "Beachten Sie" kommt ein Komma.

Reply:
Sollte "Du" sein. Siehe 36cea8b1843fc2acfdfd6a5b66f6ca408bd78883.